pod/perldelta.pod

```diff
@@ -5,35 +5,216 @@
 [ this is a template for a new perldelta file. Any text flagged as XXX needs
 to be processed before release. ]
 
-perldelta - what is new for perl v5.41.14
+perldelta - what is new for perl v5.42.0
 
 =head1 DESCRIPTION
 
-This document describes differences between the 5.41.13 release and the 5.41.14
+This document describes differences between the 5.42.0 release and the 5.40.0
 release.
 
-If you are upgrading from an earlier release such as 5.41.12, first read
-L<perl54113delta>, which describes differences between 5.41.12 and 5.41.13.
-
 =head1 Notice
 
 XXX Any important notices here
 
 =head1 Core Enhancements
 
-XXX New core language features go here. Summarize user-visible core language
-enhancements. Particularly prominent performance optimisations could go
-here, but most should go in the L</Performance Enhancements> section.
+=head2 More CORE:: subs
+
+C<chdir> has been added as a subroutine to the CORE:: namespace.
+
+Previously, code like C<&CORE::chdir($dir)> or C<< my $ref = \&CORE::chdir;
+$ref->($dir) >> would throw an error saying C<&CORE::chdir cannot be called
+directly>. These cases are now fully supported.
+
+=head2 New pragma C<L<source::encoding>>
+
+This allows you to declare that the portion of a program for the
+remainder of the lexical scope of this pragma is encoded either entirely
+in ASCII (for S<C<use source::encoding 'ascii'>>) or if UTF-8 is allowed
+as well (for S<C<use source::encoding 'utf8'>>). No other encodings are
+accepted. The second form is entirely equivalent to S<C<use utf8>>, and
+may be used interchangeably with that.
+
+The purpose of this pragma is to catch cases early where you forgot to
+specify S<C<use utf8>>.
+
+S<C<use source::encoding 'ascii'>> is automatically enabled within the
+lexical scope of a S<C<use v5.41.0>> or higher.
+
+S<C<no source::encoding>> turns off all this checking for the remainder of
+its lexical scope. The meaning of non-ASCII characters is then undefined.
+
+=head2 New C<:writer> attribute on field variables
+
+Classes defined using C<use feature 'class'> are now able to automatically
+create writer accessors for scalar fields, by using the C<:writer> attribute,
+similar to the way that C<:reader> already creates reader accessors.
+
+    class Point {
+        field $x :reader :writer :param;
+        field $y :reader :writer :param;
+    }
+
+    my $p = Point->new( x => 20, y => 40 );
+    $p->set_x(60);
+
+=head2 New C<any> and C<all> operators
+
+Two new experimental features have been added, which add two new list-processing
```

Comment:
Rather than have 2x "two new", how about "Two new experimental features add the list-processing operators, C<any> and C<all>."?

Reply:
Agreed